tools/ci_build/github/azure-pipelines/c-api-noopenmp-packaging-pipelines.yml

```diff
@@ -231,41 +173,6 @@ stages:
     buildJava: true
     buildNodejs: true
 
-#CUDA without tensorrt
-- template: templates/win-ci.yml
-  parameters:
```

Comment:
This part has been moved to `tools/ci_build/github/azure-pipelines/stages/nuget-win-cuda-packaging-stage.yml`